static/app/components/profiling/flamegraphSearch.tsx

```diff
@@ -134,14 +134,25 @@ function FlamegraphSearch({
     [canvasPoolManager]
   );
 
+  React.useEffect(() => {
+    if (typeof search.index !== 'number') {
+      return;
+    }
+
+    const frames = memoizedSortFrameResults(search.results);
```

Comment:
Any thoughts on storing the sorted results directly on `search`? That way we don't have to think about remembering to call the memorized sort and can just always access the sorted results.

Reply:
@Zylphrex Yeah, I'm just picking a fight here. The flamegraph renderer currently requires a map so that when we call draw, we just check searchresults[currentFrameBeingDrawn] and we know if we need to color it differently or not. If we sort the frames, we lose that and then need to rebuild a map here :/